initrd/bin/gui-init

```diff
@@ -430,6 +430,7 @@ show_options_menu()
     --menu "" 0 80 10 \
     'b' ' Boot Options -->' \
     't' ' TPM/TOTP/HOTP Options -->' \
+    'h' ' Change system time' \
```

Comment:
Otherwise the two entry points show different behavior: one with datestamp one without. change-time should be where date reminder occur IMO